viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -211,7 +213,10 @@ export class ComboControls extends EventDispatcher {
 
     _spherical.makeSafe();
     _camera.position.setFromSpherical(_spherical).add(_target);
-    _camera.lookAt(this.lookAtViewTarget ? this._viewTarget : _target);
+
+    if (this._rawCameraRotation.equals(new Quaternion()))
```

Comment:
Avoid `new` here - it's in the hot path and we should be very careful about allocations. Declare a variable outside the function or see if there is some other way of doing the same check without allocation.

Reply:
Also, add `{ ... }` for the if-else clauses

Reply:
Create the `isIdentityQuaternion` function and check using it